x/nexus/keeper/tx_id_generator.go

```diff
@@ -0,0 +1,26 @@
+package keeper
```

Comment:
I would actually call this type MsgIdGenerator. It's not really generating a tx id.